app/models/topic.rb

```diff
@@ -265,6 +265,19 @@ def from_email_address
     end
   end
 
+  def check_attachments
```

Comment:
I feel that maybe this should be renamed to `attachments?`

Reply:
yeah. can, suddenly i feel its better to renamed it.